### PR TITLE
HII-36: a few styling changes.:

### PR DESCRIPTION
**Pull request by n3op2**
## What

A few styling changes:
- label
- badge
- use of HX rather than h3

### Screenshots
Screenshot 2023-01-10 at 10 54 11

> different fonts
Screenshot 2023-01-10 at 11 06 42
Screenshot 2023-01-10 at 11 06 30

### CODE REVIEW

File: src/Dropdown/index.tsx [hunk 1]
```diff
@@ -100,14 +108,14 @@ const HiiMultiSelect: IDropdown = ({ onChange, value = [], ...props }) => {
...
     const removedValue = value.find((val: any) => val.value === name)
 
     if (!removedValue) return null
-    props.update(value)
 
     onChange(
-      value.filter((val: any) => val.value !== name),
-      { name, action: 'remove-value', removedValue }
+      value.filter((val: any) => val.value !== name)
+      // { name, action: 'remove-value', removedValue }
```
Comment (jonmattgray) [inline src/Dropdown/index.tsx:118]: comment?

Comment (n3op2) [reply src/Dropdown/index.tsx]: yes, added some context

File: src/Dropdown/index.tsx [hunk 2]
```diff
@@ -100,14 +108,14 @@ const HiiMultiSelect: IDropdown = ({ onChange, value = [], ...props }) => {
   const handleRemoveValue = (e: any) => {
     if (!onChange) return null
     const { name } = e.currentTarget
+    console.log(name)
```
[the remark this answers is not in the document]
Comment (n3op2) [reply src/Dropdown/index.tsx]: done

File: src/Dropdown/index.tsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import React from 'react'
 import Select, { StylesConfig, components } from 'react-select'
 import styled from 'styled-components'
+import { HX } from '../Heading/index.js'
```
Comment (n3op2) [inline src/Dropdown/index.tsx:4]: using .js import as it's being used in section component

**Pull request merged**